src/search/_base.scss

```
@@ -28,16 +28,27 @@
 		vertical-align: middle;
 	}
 
+	.submit {
```

did you mean: `[type=submit]` ? the class .submit don't seem to be used

---

The class is used on the parent of the button and it is the only way to distinguish it, so it is needed. However, I find the name of the class too broad. It should be called something like: `wb-srch-sub`, like its child ID.
```suggestion
	.wb-srch-sub{
```